cves/kernel/CVE-2016-2549.yml

```diff
@@ -55,7 +55,13 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```

Comment:
What does this vulnerability cause, i.e. DOS?

Reply:
This results from the deadlock when hrtimer_cancel() is called inside its callback function.